client/src/components/DashboardCoponent/ProfileCard.tsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import LocationOnIcon from '@material-ui/icons/LocationOn';
+import { CardActionArea, CardActions, CardContent, CardMedia, Typography, Card, Divider } from '@material-ui/core';
+import Rating from '@material-ui/lab/Rating';
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 345,
+  },
+  content: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  media: {
+    height: 140,
+    width: 140,
+    borderRadius: '50%',
+    margin: '10px',
+  },
+  name: {
+    justifyContent: 'center',
+  },
+});
+
+function ProfileCard() {
+  const [value, setValue] = React.useState<number | null>(1);
+  const classes = useStyles();
+  return (
+    <Card className={classes.root}>
```

Comment:
In the future, you can create an object with all the same fields as the response data you will receive rather than hardcoding information, that will make life easy for you :)